features/environment.py

```diff
@@ -599,9 +599,12 @@ def _install_uat_in_container(
     """
     cmds = []
     if not deb_paths:
-        deb_names = (
-            " ".join(UA_DEBS) if cloud_api else "ubuntu-advantage-tools"
+        pkg_names = (
+            [deb.replace(".deb", "") for deb in UA_DEBS]
+            if cloud_api
```

Comment:
I don't think this is quite correct because cloud_api is for both awsgeneric and awspro test targets. We don't want to add ubuntu-advantage-pro deb on awsgeneric or azuregeneric machine_types